src/tests/components/CourseDetails.test.js

```diff
@@ -3,6 +3,19 @@ import React from "react";
 import { CourseDetails } from "../../components/CourseDetails";
 
 describe("Course details", () => {
+    it("displays award tag", () => {
+        const exampleCourse = {
+            award: "BA (Hons)",
+        };
+
+        render(<CourseDetails course={exampleCourse} />);
+
+        const tags = screen.getAllByRole("listitem");
+
+        expect(tags[0]).toHaveTextContent("BA (Hons)");
+        expect(within(tags[0]).getByTestId("tag-icon")).toHaveClass("c-icon--bank");
```

Comment:
Probably don't need to use `getAllByRole` here given that there's only one `listitem` role?
```suggestion
        const tag = screen.getByRole("listitem");

        expect(tag).toHaveTextContent("BA (Hons)");
        expect(within(tag).getByTestId("tag-icon")).toHaveClass("c-icon--bank");
```

Reply:
Corrected in Mark's branch, so won't do here.

Reply:
Mark's branch has been merged (I think). Also isn't this a separate test from the one he wrote?